keps/prod-readiness/sig-node/34.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 34
+stable:
+  approver: "@johnbelamaric"
```

Comment:
ok, it's on my list, I will keep checking for updates...

Reply:
Should be good to review now, I'll try to be as responsive as I can for requested updates. This PR has been sitting in beta since 1.11 and we'd like to graduate it this release if possible; there isn't much work to be done.